config/rips.ini

```diff
@@ -15,11 +15,10 @@ sp.unserialize_hmac.enable();
 # AbanteCart 1.2.8 - Multiple SQL Injections
 # https://blog.ripstech.com/2016/abantecart-multiple-sql-injections/ 
 sp.disable_function.function("ALanguageManager::_clone_language_rows").filename("/core/lib/language_manager.php").param("from_language").value_r("[^0-9]").drop();
-sp.disable_function.function("echo").filename("/static_pages/index.php").var("_SERVER[HTTP_HOST]").value_r("[^0-9a-z-.]").drop();
+sp.disable_function.function("define").filename("/static_pages/index.php").var("_SERVER[PHP_SELF]").value_r("[^0-9a-z\/\=?._-]").drop();
```

Comment:
The filename is wrong, the value must be the path of the file on the filesystem, and not the path in the document root.
filename_r() should be used instead (something like this should work : `.filename_r("/static_pages/index.php$")`)